dev-tools/commands/src/upload-track.mjs

```diff
@@ -92,8 +93,7 @@ program.command("upload-track")
           premium_conditions: premiumConditions ? JSON.parse(premiumConditions) : null,
           ai_attribution_user_id: null
         },
-        () => null,
-        /* writeMetadataThroughChain */ true
+        () => null
```

Comment:
we probably want this writeMetadataThroughChain flag still right? or is that encompassed by this method already

Reply:
it's encompassed! v2 only writes through chain